package.json

```
@@ -1,11 +1,14 @@
 {
   "name": "cwp-starter-theme",
-  "version": "3.0.0",
+  "version": "4.0.0",
   "private": true,
+  "engines": {
+    "node": "^18.x"
+  },
   "scripts": {
-    "watch": "webpack --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js --watch",
-    "dev": "yarn lint && webpack --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
-    "build": "yarn lint && NODE_ENV=production webpack --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
+    "watch": "webpack --progress --config=node_modules/laravel-mix/setup/webpack.config.js --watch",
+    "dev": "yarn lint && webpack --progress --config=node_modules/laravel-mix/setup/webpack.config.js",
+    "build": "yarn lint && NODE_ENV=production webpack --progress --config=node_modules/laravel-mix/setup/webpack.config.js",
```

We should probably use the new `mix` command if we're using laravel mix, as per [the upgrade guide](https://github.com/laravel-mix/laravel-mix/blob/master/UPGRADE.md#update-your-npm-scripts)
```suggestion
    "watch": "mix watch",
    "dev": "mix",
    "build": "mix --production",
```

---

Updated